api/proto/dop/issue/sync/sync.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package erda.dop.issue.sync;
+option go_package = "github.com/erda-project/erda-proto-go/dop/issue/sync/pb";
+import "google/api/annotations.proto";
+import "common/openapi.proto";
+
+service IssueSyncService {
+  option(erda.common.openapi_service) = {
+    service : "dop",
+    auth : {
+      check_login : true,
+      check_token: true,
+    }
+  };
+
+  rpc IssueSync(IssueSyncRequest) returns(IssueSyncResponse) {
+    option(google.api.http) = {
+      post : "/api/issues/actions/sync",
```

Comment:
I think `patch-update` is better to understand, what do you think?

Reply:
and define a dedicated api is more helpful to understand: `/api/issues/actions/patch-update-labels`.

because it's hard to unify other updates.

Reply:
and issueID should in path: `/api/issues/{ID}/actions/patch-update-labels`

Reply:
It is a subset of batch-update-issue, not on a specific issue,  what about api/issues/actions/batch-update-children

Reply:
Changed to api/issues/{id}/actions/sync-update-fields-to-children